### PR TITLE
corrects mis-capitalized instances of scaffold in code snippets

### PR DESCRIPTION
**Pull request by nellshamrell**
Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/blog/2017-10-09-node-scaffolding-habitat.html.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ This creates a habitat directory along with several starter files.  Open up habi
 pkg_name=misfit_toys
 pkg_origin=your_origin
 pkg_version="0.1.0"
-pkg_Scaffolding="core/Scaffolding-node"
+pkg_Scaffolding="core/scaffolding-node"
```
Comment (cnunciato) [inline www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: `pkg_scaffolding`

Comment (nellshamrell) [reply www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: ty, fixing.

Comment (nellshamrell) [reply www/source/blog/2017-10-09-node-scaffolding-habitat.html.md]: fixed!

**Comment by tashimi** [comment]
@thesentinels approve 

tenor-11272351

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/340654610).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to pass the test suite](https://travis-ci.org/habitat-sh/habitat/builds/340654610).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/340654610) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/340654610).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/340654610).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**